### PR TITLE
Remove the remains of UploadSet

### LINKED ISSUE #1775: Remove the remnants of UploadSets
They aren't used anymore.

### PR DESCRIPTION
**Pull request by jcoyne**
@projecthydra/sufia-code-reviewers

UploadSet is no longer useful object to keep around.

Fixes #1775

### CODE REVIEW

File: spec/controllers/works_controller_spec.rb [hunk 1]
```diff
@@ -1,87 +0,0 @@
-require 'spec_helper'
```
Comment (mjgiarlo) [inline spec/controllers/works_controller_spec.rb]: Why dump this file?

File: spec/tasks/rake_spec.rb [hunk 2]
```diff
@@ -1,77 +0,0 @@
...
-      it { is_expected.to start_with("Output: empty-upload_set contains no files - to delete, rerun with the remove option") }
-
-      describe "removing the empty upload_set" do
-        subject { run_task "sufia:empty_upload_sets", "remove" }
-        it { is_expected.to start_with("Output: empty-upload_set contains no files - deleted") }
-      end
-    end
-  end
-
-  describe "sufia:migrate" do
```
Comment (mjgiarlo) [inline spec/tasks/rake_spec.rb]: Why are we removing stuff from here down?

File: spec/controllers/works_controller_spec.rb (hunk 1 above)
Comment (jcoyne) [inline spec/controllers/works_controller_spec.rb]: It was testing the same things/ways as `spec/controllers/generic_works_controller_spec.rb` It seemed silly to have two.

File: spec/tasks/rake_spec.rb (hunk 2 above)
Comment (jcoyne) [inline spec/tasks/rake_spec.rb]: That was an oversight.

File: spec/controllers/works_controller_spec.rb (hunk 1 above)
Comment (mjgiarlo) [inline spec/controllers/works_controller_spec.rb]: Yes, that is quite silly.

File: spec/tasks/rake_spec.rb (hunk 2 above)
Comment (mjgiarlo) [inline spec/tasks/rake_spec.rb]: OK, waiting on a new push.

**Comment by mjgiarlo** [comment]
@jcoyne will be restoring some stuff in `rake_spec.rb`. Please wait on that before merging.

**Comment by jcoyne** [comment]
The stuff from rake_spec.rb has been restored.

**Comment by mjgiarlo** [comment]
:eyes: Travis

**Pull request merged**